operators/pkg/controller/remotecluster/labels.go

```diff
@@ -14,6 +14,8 @@ import (
 const (
 	// RemoteClusterDynamicWatchesFinalizer designates a finalizer to clean up unused watches.
 	RemoteClusterDynamicWatchesFinalizer = "dynamic-watches.remotecluster.k8s.elastic.co"
+	// RemoteClusterSeedServiceFinalizer designates a finalizer to clean up a seed Service.
+	RemoteClusterSeedServiceFinalizer = "seed-service.remotecluster.k8s.elastic.co"
```

Comment:
Side note: core object finalizers  require a `/` to be fully qualified to my knowledge, as in `remotecluster.k8s.elastic.co/seed-svc` This is being attached to a CRD so it should be fine. I wonder if we should standardize nevertheless, maybe outside the scope of this PR though.

Reply:
Should we create a separate issue for this? I don't see a slash in any other finalizers we use atm.